### ISSUE TITLE
`fedimint-bin-tests` should output PIDs of started processes

### ISSUE THREAD

**Issue by elsirion** [fedimint/fedimint#2274]
Having the PIDs would allow killing and restarting single processes manually to simulate faults.

**Comment by Maan2003** [comment]
by printing do you mean just in logs? or in some specific format on stdout.

we also need to print environment variables for the command. the printed command should pastable into bash directly. (use single quotes for arguments)

**Comment by elsirion** [comment]
> by printing do you mean just in logs?

Yes, that would help I assume, unless me killing the process kills all other processes. What I want to be able to do is "start all `fedimintd`s but one" and then start the last manually or not at all.

**Comment by justinmoon** [comment]
We need is some way to stop / restart child processes. Perhaps we could just print out PIDs and manually kill them. Perhaps eventually come CLI command to help with this.
